generators/client/templates/react/jest.conf.js.ejs

```diff
@@ -24,10 +23,6 @@ module.exports = {
   testPathIgnorePatterns: [
     '<rootDir>/node_modules/'
   ],
-  setupFiles: [
-    '<rootDir>/<%= CLIENT_TEST_SRC_DIR %>spec/icons-mock.ts',
-    '<rootDir>/<%= CLIENT_TEST_SRC_DIR %>spec/storage-mock.ts'
-  ],
```

Comment:
If preserving `icons-mock.ts` as `setup-tests.ts` then this section should be:
```
setupFiles: ['<rootDir>/src/main/webapp/app/setup-tests.ts'],
```

Reply:
ok